src/python/pants/reporting/reporting.py

```diff
@@ -44,6 +45,10 @@ def register_options(cls, register):
              help='Controls the printing of workunit tool output to the console. Workunit types are '
                   '{workunits}.  Possible formatting values are {formats}'.format(
                workunits=list(WorkUnitLabel.keys()), formats=list(ToolOutputFormat.keys())))
+    register('--zipkin-endpoint', advanced=True, default=None,
+              help='Enables Zipkin reporter and sets the endpoint')
+    register('--zipkin-encoding', choices=['json', 'x-thrift'], advanced=True, default='x-thrift',
```

Comment:
Let's accept `thrift` on the command line, and set the header to `x-thrift` in our code if we need to; `x-thrift` is a strange command line interface to be exposing :)